core/pkg/api/grpc/task.go

```
@@ -70,25 +71,35 @@ var (
 )
 
 func translateTaskForward(m *api.Task) *gapi.Task {
-	return &gapi.Task{
+	gt := &gapi.Task{
 		Key:      uint64(m.Key),
 		Name:     m.Name,
 		Type:     m.Type,
 		Config:   m.Config,
 		Internal: m.Internal,
 		Snapshot: m.Snapshot,
 	}
+	if m.Status != nil {
+		gt.Status, _ = status.TranslateToPB[task.StatusDetails](status.Status[task.StatusDetails](*m.Status))
```

shouldn't ignore error